File: app/src/main/java/com/letianpai/robot/control/mode/parser/RobotModeItem.java
package com.letianpai.robot.control.mode.parser;

public class RobotModeItem {

    private int priority;

    private int modeId;

    private boolean isElectricityOn;

    public int getModeId() {
        return modeId;
    }

    public int getPriority() {
        return priority;
    }

    public boolean isElectricityOn() {
        return isElectricityOn;
    }

    public void setElectricityOn(boolean electricityOn) {
        isElectricityOn = electricityOn;
    }

    public void setModeId(int modeId) {
        this.modeId = modeId;
    }

    public void setPriority(int priority) {
        this.priority = priority;
    }

    @Override
    public String toString() {
        return "{" +
                "priority:" + priority +
                ", modeId:" + modeId +
                ", isElectricityOn:" + isElectricityOn +
                '}';
    }
}

File: app/src/main/java/com/letianpai/robot/control/mode/parser/RobotModeItem.kt
package com.letianpai.robot.control.mode.parser

class RobotModeItem {
    var priority: Int = 0

    var modeId: Int = 0

    var isElectricityOn: Boolean = false

    override fun toString(): String {
        return "{" +
                "priority:" + priority +
                ", modeId:" + modeId +
                ", isElectricityOn:" + isElectricityOn +
                '}'
    }
}
